Nfield.Quota.Tests/QuotaFrameValidatorTests.cs

```diff
@@ -234,7 +234,7 @@ public void Definitions_CannotHaveInvalidOdinVariableName()
                 "and cannot be empty." +
                 " They can only ​start with​ a letter. First character cannot be '_' or a number." +
                 " Variable definition Id '{0}' with name '{1}' has an invalid Odin Variable Name '{2}'",
-                varId, variableName, invalidOdinVarName);
+                varId, variableName, invalidOdinVarName).Replace("\u200b", "");
```

Comment:
Why this change? Looks unrelated to story.

Reply:
@joukevandermaas @mvdr do you know anything about this?
I changed `QuotaFrameValidator.cs` to remove the non-printable character, and @krakie changed the test accordingly. However since the test was like this, maybe there was a reason the code was like this?

Reply:
#102

Reply:
I removed some invisible spaces from the strings. They don't show up in the diff (because they're invisible, duh!).
I now also undid this change to the test and instead removed some more invisible space in the tests.

Reply:
Well, invisible spaces and random characters should not be allowed in odin variable names. So I think that's what the test was testing. I don't really think it should be changed, but you can use escape characters instead if you think that is better.

Reply:
I don't think that's the case. They were just at random places in the message, eg "They can only ​start with​ &lt;invisible space&gt;a letter".